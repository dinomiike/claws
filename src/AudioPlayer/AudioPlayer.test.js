import React from 'react'
import renderer from 'react-test-renderer'
import { AudioPlayer } from '../'

it('renders an AudioPlayer correctly', () => {
  const tree = renderer.create(<AudioPlayer />).toJSON()
  expect(tree).toMatchSnapshot()
})
