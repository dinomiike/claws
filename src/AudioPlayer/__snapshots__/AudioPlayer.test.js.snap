// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an AudioPlayer correctly 1`] = `
<div
  className="AudioPlayer"
>
  <h1>
    Audio Player
  </h1>
  <div>
    Moo
  </div>
</div>
`;
